src/pages/ActivationPage.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Route } from 'react-router-dom'
+
+import { render } from '../../test/testUtils'
+
+import ActivationPage from './ActivationPage'
+
+// https://www.npmjs.com/package/jest-fetch-mock
```

Comment:
nit: delete comment.

Reply:
the comment was to explain the approach, which isn't obvious, I think.